Makefile

```diff
@@ -33,10 +33,10 @@ deps:
 	git submodule add git@github.com:ZeroSync/ZeroSync.git
 
 deps-default-version:
-	pip install ecdsa fastecdsa sympy cairo-lang==0.10.3 maturin
+	pip install ecdsa fastecdsa sympy cairo-lang==0.10.3 maturin typeguard==2.13.0
 	python3 -m venv cairo-rs-py-env
 	. cairo-rs-py-env/bin/activate && \
-	pip install cairo_lang==0.10.3 && \
+	pip install cairo-lang==0.10.3 typeguard==2.13.0 && \
```

Comment:
why not 0.11?

Reply:
This PR is not the 0.11 update, its a previous update that wasnt merged